Library/Formula/apcupsd.rb

```diff
@@ -0,0 +1,106 @@
+class Apcupsd < Formula
+  desc "Daemon for controlling APC UPSes"
+  homepage "http://www.apcupsd.org"
+  url "https://downloads.sourceforge.net/project/apcupsd/apcupsd%20-%20Stable/3.14.13/apcupsd-3.14.13.tar.gz"
+  sha256 "57ecbde01d0448bf8c4dbfe0ad016724ae66ab98adf2de955bf2be553c5d03f9"
+
+  option "with-usb", "Build with support for USB UPSes."
+
+  depends_on "gd"
+  depends_on "libusb-compat"
+
+  def install
+    cd "src/apcagent" do
+      # Install apcagent.app to `prefix` instead of /Applications.
+      inreplace "Makefile", "Applications", "#{"#{prefix}".gsub(%r{/^\//}, "")}"
+    end
+
+    sysconfdir = "#{etc}/apcupsd"
+
+    cd "platforms/darwin" do
+      # Install binaries to `prefix`.
+      # Patch submitted to upstream repo:
+      #   https://sourceforge.net/p/apcupsd/mailman/message/34627459/
+      inreplace "Makefile", "/sbin", "$(sbindir)"
+
+      # Install the LaunchDaemon and kernel extension to `prefix`.
+      inreplace "Makefile", "/Library/LaunchDaemons", "#{prefix}/Library/LaunchDaemons"
+      inreplace "Makefile", "/System/Library/Extensions", kext_prefix
+
+      # Tell the launch script and LaunchDaemon to use the appropriate sbin and config directories.
+      inreplace "apcupsd-start", "/sbin", opt_sbin
+      inreplace "apcupsd-start", "/etc/apcupsd", sysconfdir
+      inreplace "org.apcupsd.apcupsd.plist", "/sbin", opt_sbin
+      inreplace "org.apcupsd.apcupsd.plist", "/etc/apcupsd", sysconfdir
+
+      # Custom uninstaller is not needed as everything is installed to `prefix`.
+      inreplace "Makefile", /.*apcupsd-uninstall.*/, ""
+    end
```

Comment:
Can you please submit these patches to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!